config/deploy/aws.rb

```diff
@@ -5,7 +5,7 @@
 # property set.  Don't declare `role :all`, it's a meta role.
 set :stage, :aws
 set :rails_env, 'production'
-server 'ucsb.curationexperts.com', user: 'deploy', roles: [:web, :app, :db, :resque_pool]
+server 'ucsb.curationexperts.com', user: 'adrl', roles: [:web, :app, :db, :resque_pool]
```

Comment:
Yes, thanks. You can leave the others, but I want to be sure we can deploy this week uninterrupted on the existing sandbox.